parlai/crowdsourcing/tasks/pairwise_per_turn_eval/utils.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.crowdsourcing.utils.tests import AbstractParlAIChatTest
+
+import unittest
+from typing import Any, Dict, Optional, Tuple
+
+
+class AbstractPerTurnEvalTest(AbstractParlAIChatTest, unittest.TestCase):
```

Comment:
Ugh I know I said that we should move this file, but looking at it now, since it's not yet functional honestly I'd probably keep it where it was until we get it working -__-